### ISSUE TITLE
Add public keys for more issuers

### ISSUE THREAD

**Issue by kentalee** [mrautio/emvpt#12]
EFTLab provides a list of keys from well-known issuers:
https://www.eftlab.com/knowledge-base/243-ca-public-keys/

Wish it could be added to the default configuration. Thanks a lot.

**Comment by mrautio** [comment]
I haven't added the keys in the past as I don't have test cards to verify if the cards work with the project. However, if there's need for having more keys in the configuration I (or anyone else) could add them easily.

Currently added keys are stored and can be edited in following files:
- https://github.com/mrautio/emvpt/blob/master/emvpt/src/config/scheme_ca_public_keys_production.yaml
- https://github.com/mrautio/emvpt/blob/master/emvpt/src/config/scheme_ca_public_keys_test.yaml

**Comment by kentalee** [comment]
Thank you for replying. May I later issue a PR to add keys that can successfully complete the transaction on test card?

**Comment by mrautio** [comment]
Yes, you may. 😄  Let me know if any help is needed in case of any issues.
